src/compute-types/src/explain/text.rs

```diff
@@ -212,7 +225,11 @@ impl DisplayText<PlanRenderingContext<'_, Plan>> for Plan {
             } => {
                 let exprs = mode.seq(exprs, None);
                 let exprs = CompactScalars(exprs);
-                writeln!(f, "{}FlatMap {}({})", ctx.indent, func, exprs)?;
+                writeln!(
+                    f,
+                    "{}FlatMap {}({}){}",
+                    ctx.indent, func, exprs, annotations
+                )?;
```

Comment:
Nit, here and elsewhere to make this more readable:
```suggestion
                writeln!(
                    f,
                    "{}FlatMap {func}({exprs}){annotations}",
                    ctx.indent
                )?;
```

Reply:
I'd like to leave that as a follow up, since it's a large-ish refactoring that warrants a separate PR.